### PR TITLE
Use locale.exe instead of wmic on Windows

### LINKED ISSUE #35: os-locale calls wmic with no extension and without explicit path
Our Windows app that uses os-locale has been flagged as a security risk because it will launch any file named "wmic", regardless of extension, and the file can be anywhere on the PATH, including the app's own directory. It could be a vbs file, for example.

The suggested secure way (if you have to use wmic):

1. Always use the exe extension.
2. Always use the full standard path to wmic.exe.

### PR DESCRIPTION
**Pull request by MarkTiedemann**
Fixes #35

### CODE REVIEW

**Comment by MarkTiedemann** [comment]
Added an `.appveyor.yml`, but no badge yet.

**Comment by sindresorhus** [comment]
I would prefer just using Travis if possible. They support Windows now.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict and replace AppVeyor with Travis?

File: index.js [hunk 1]
```diff
@@ -68,17 +68,13 @@ function getUnixLocaleSync() {
 }
 
 function getWinLocale() {
-	return execa.stdout('wmic', ['os', 'get', 'locale'])
-		.then(stdout => {
-			const lcidCode = parseInt(stdout.replace('Locale', ''), 16);
-			return lcid.from(lcidCode);
-		});
+	return execa.stdout(path.join(__dirname, 'locale.exe'))
+		.then(stdout => stdout.replace(/-/, '_'));
```
Comment (sindresorhus) [inline index.js]: The `replace` here is no longer needed: https://github.com/sindresorhus/os-locale/commit/2a3fccb952db4ba64df025dc04a5467a6d42cf42

**Comment by MarkTiedemann** [comment]
Sorry for the late response, @sindresorhus.

I don't think this should be merged. In fact, I'll close this PR as well as other similar PRs for now.

`locale.exe` only works on `x64`, not `x86` (`ia32`). It's also far slower than a native module. Plus my implementation in this PR doesn't provide proper fallbacks.

As I said in a different thread, I'm working on a good (pre-)build process / structure for Windows Node native modules. As soon as I have finished with that, I'll properly (re-)implemented the respective native modules, and create new PRs for your modules.

I'll ping you once I'm ready.

PS: Please keep pinging me in regards to module ideas. These great ideas are the reason I'm doing this. :)

**Pull request closed without merging**